mesonbuild/modules/hotdoc.py

```diff
@@ -357,7 +357,7 @@ def make_targets(self) -> T.Tuple[HotdocTarget, mesonlib.ExecutableSerialisation
 
         install_script = None
         if install:
-            datadir = os.path.join(self.state.get_option('prefix'), self.state.get_option('datadir'))
+            datadir = os.path.join(self.stateoptstore.get_value_for('prefix'), self.stateoptstore.get_value_for('datadir'))
```

Comment:
```suggestion
            datadir = os.path.join(self.state.optstore.get_value_for('prefix'), self.state.optstore.get_value_for('datadir'))
```

Reply:
or more probably self.state.environment.coredata.optstore...

Reply:
Those should have been left alone, I don't have hotdoc on my system and it shows.

Eventually I want to get rid of the `ModuleState.get_option` too, just haven't gotten there yet.